CONTENTS.md

```diff
@@ -411,7 +416,7 @@ Verifies 32bit binaries were imported properly towards a project.
 * Sources: [scan_baselibs.py](scan_baselibs.py)
 * Documentation: --
 * Package: openSUSE-release-tools
-* Usage: ???
+* Usage: obsolete
```

Comment:
We may mention that this apparently became obsolete once this fix was introduced in OBS https://github.com/openSUSE/open-build-service/pull/7662